tools/genesis/src/wizard.rs

```diff
@@ -462,6 +450,32 @@ pub fn what_host() -> Result<HostAndPort, anyhow::Error> {
     Ok(ip)
 }
 
+pub fn initialize_validator_configs(data_path: &PathBuf, github_username: Option<&str>) -> Result<(), anyhow::Error> {
+    let to_init = Confirm::new()
+        .with_prompt(format!(
+            "Want to freshen configs at {:?} now?",
+            data_path.clone()
+        ))
+        .interact()?;
+    if to_init {
+        let host = what_host()?;
+
+        let keep_legacy_address = Confirm::new()
+            .with_prompt("Is this a legacy V5 address you wish to keep?")
+            .interact()?;
+
+        initialize_host(
```

Comment:
I think this function `initialize_host` actually would belong in the tools/config module instead of genesis. I think that's an easy change.
The imports make more sense to me if genesis tools automation is driving the lower level config library.

Reply:
Like the idea. I created a new module called `host` under `config` which will be the new place for `initialize_host`. How's that?
I',m also thinking about moving `node_yaml` from `genesis` to under `config`, what do you think?

Reply:
all good changes!